pkg/lib/form-layout.less

```diff
@@ -43,8 +47,17 @@
 // a <form class="ct-form-layout"> and layout magic happens.
 
 .ct-form-layout {
+  // Locally redefine padding back to Bootstrap values
+  @padding-large-vertical: 10px;
+  @padding-small-vertical: 5px;
+  @padding-small-horizontal: 10px;
+  // Bootstrap & PatterFly use a 1px border around widgets
+  @border-width: 1px;
```

Comment:
Border-width is defined as a LESS variable to make the @widget-height calculation a bit more descriptive.

Since it compiles down to normal CSS anyway, there's absolutely no harm whatsoever to have a single-use variable in LESS.